2021-05-18 11:24:02,200 INFO    MainThread:173586 [wandb_setup.py:_flush():69] setting env: {}
2021-05-18 11:24:02,200 INFO    MainThread:173586 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-18 11:24:02,200 INFO    MainThread:173586 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/American-Sign-Language-Recognition/wandb/run-20210518_112402-3kmqzsrw/logs/debug.log
2021-05-18 11:24:02,200 INFO    MainThread:173586 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/American-Sign-Language-Recognition/wandb/run-20210518_112402-3kmqzsrw/logs/debug-internal.log
2021-05-18 11:24:02,200 INFO    MainThread:173586 [wandb_init.py:init():369] calling init triggers
2021-05-18 11:24:02,201 INFO    MainThread:173586 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-18 11:24:02,201 INFO    MainThread:173586 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 1r6cfx2e
2021-05-18 11:24:02,202 INFO    MainThread:173586 [wandb_run.py:finish():1208] finishing run ranuga-d/Sign-Language-Recognition/1r6cfx2e
2021-05-18 11:24:02,208 INFO    MainThread:173586 [jupyter.py:save_history():349] saving 42 cells to _session_history.ipynb
2021-05-18 11:24:02,217 INFO    MainThread:173586 [wandb_run.py:_config_callback():787] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2021-05-18 11:24:02,240 INFO    MainThread:173586 [jupyter.py:save_ipynb():268] looking for notebook: main.ipynb
2021-05-18 11:24:03,196 INFO    MainThread:173586 [wandb_init.py:_jupyter_teardown():267] saved code and history
2021-05-18 11:24:03,196 INFO    MainThread:173586 [wandb_init.py:_jupyter_teardown():268] cleaning up jupyter logic
2021-05-18 11:24:03,196 INFO    MainThread:173586 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 0
2021-05-18 11:24:03,196 INFO    MainThread:173586 [wandb_run.py:_restore():1507] restore
2021-05-18 11:24:08,198 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: None
2021-05-18 11:24:11,380 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 94486
  total_bytes: 94486
}

2021-05-18 11:24:12,442 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 94486
  total_bytes: 94486
}

2021-05-18 11:24:12,685 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 94486
  total_bytes: 132622
}

2021-05-18 11:24:14,088 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 132622
  total_bytes: 132622
}

2021-05-18 11:24:14,190 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 132622
  total_bytes: 132622
}

2021-05-18 11:24:14,291 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 132622
  total_bytes: 132622
}

2021-05-18 11:24:14,392 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 132622
  total_bytes: 132622
}

2021-05-18 11:24:14,494 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 132622
  total_bytes: 132622
}

2021-05-18 11:24:14,595 INFO    MainThread:173586 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  artifact_count: 3
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 132622
  total_bytes: 132622
}

2021-05-18 11:24:15,888 INFO    MainThread:173586 [wandb_run.py:_show_files():1879] logging synced files
2021-05-18 11:24:15,891 INFO    MainThread:173586 [wandb_init.py:init():418] starting backend
2021-05-18 11:24:15,892 INFO    MainThread:173586 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-18 11:24:15,894 INFO    MainThread:173586 [backend.py:ensure_launched():132] starting backend process...
2021-05-18 11:24:15,930 INFO    MainThread:173586 [backend.py:ensure_launched():137] started backend process with pid: 189707
2021-05-18 11:24:15,930 INFO    MainThread:173586 [wandb_init.py:init():423] backend started and connected
2021-05-18 11:24:15,941 INFO    MainThread:173586 [wandb_init.py:init():465] updated telemetry
2021-05-18 11:24:15,941 INFO    MainThread:173586 [wandb_init.py:init():484] communicating current version
2021-05-18 11:24:17,107 INFO    MainThread:173586 [wandb_init.py:init():489] got version response 
2021-05-18 11:24:17,107 INFO    MainThread:173586 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-18 11:24:18,057 INFO    MainThread:173586 [wandb_init.py:init():522] starting run threads in backend
2021-05-18 11:24:23,063 INFO    MainThread:173586 [wandb_run.py:_console_start():1565] atexit reg
2021-05-18 11:24:23,063 INFO    MainThread:173586 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.WRAP
2021-05-18 11:24:23,063 INFO    MainThread:173586 [wandb_run.py:_redirect():1455] Wrapping output streams.
2021-05-18 11:24:23,064 INFO    MainThread:173586 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-18 11:24:23,064 INFO    MainThread:173586 [wandb_init.py:init():546] run started, returning control to user process
2021-05-18 11:25:22,163 INFO    MainThread:173586 [wandb_setup.py:_flush():69] setting env: {}
2021-05-18 11:25:22,163 INFO    MainThread:173586 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-18 11:25:22,163 INFO    MainThread:173586 [wandb_init.py:_log_setup():336] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/American-Sign-Language-Recognition/wandb/run-20210518_112522-3vbd81mz/logs/debug.log
2021-05-18 11:25:22,163 INFO    MainThread:173586 [wandb_init.py:_log_setup():337] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/CNN/American-Sign-Language-Recognition/wandb/run-20210518_112522-3vbd81mz/logs/debug-internal.log
2021-05-18 11:25:22,164 INFO    MainThread:173586 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fdfa7dbdf28>
2021-05-18 11:25:22,164 INFO    MainThread:173586 [wandb_init.py:init():369] calling init triggers
2021-05-18 11:25:22,164 INFO    MainThread:173586 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-18 11:25:22,164 INFO    MainThread:173586 [wandb_init.py:init():391] re-initializing run, found existing run on stack: 3kmqzsrw
2021-05-18 11:25:22,165 INFO    MainThread:173586 [wandb_run.py:finish():1208] finishing run ranuga-d/Sign-Language-Recognition/3kmqzsrw
